scripts/policy/frameworks/intel/seen/file-names.zeek

```diff
@@ -3,9 +3,28 @@
 
 event file_new(f: fa_file)
 	{
+	# If there are connections attached, we'll be using
+	# file_over_new_connection() for reporting the
+	# filename instead as it's more likely to be populate.
```

Comment:
```suggestion
	# filename instead as it's more likely to be populated.
```

Reply:
Fixed and also moved the tests into scripts/policy.